doc/source/examples/dem/bunny-drill/bunny-drill.rst

```diff
@@ -134,7 +134,7 @@ The time end of the simulation is 4.75 seconds after which the bunny has done on
 Solid Objects (Drilling)
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The bunny is defined using the solid objects feature of Lethe. The surface mesh of the bunny is a GMSH file. The translational of the velocity is defined to have a periodic motion along the axis of the cylinder and the bunny is rotating at a constant angular velocity. This complex drilling motion is fully parametrized from the input file using the function parser functionalities of the translational and the angular velocity of the solid object.
+The bunny is defined using the solid objects feature of Lethe. The surface mesh of the bunny is a GMSH file. The translational of the velocity is defined to have a periodic motion along the axis of the cylinder and the bunny is rotating at a constant angular velocity once the particles have been loaded (:math:`t>2\text{s}`) . This complex drilling motion is fully parametrized from the input file using the function parser functionalities of the translational and the angular velocity of the solid object.
```

Comment:
I think it is going down and up and down etc. @acdaigneault

Reply:
The example is not long enough to see that

Reply:
simulation end time should higher

Reply:
I mean a sinusoidal is a periodic motion. I can remove the mention if you prefer.
I don't want to increase the end time since I don't think there's any point in seeing two motions.